scripts/generate_dataset.py

```diff
@@ -16,7 +16,7 @@
 from core.controller import Controller
 from systems import add_system_arguments, get_system
 
-parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(conflict_handler='resolve')
```

Comment:
is this because there are multiple --verbose?

Reply:
Yes, that is exactly right.  There is a `verbose` in generate_dataset and one in neural/evalutor.  Removing the `verbose` in generate_dataset causes other sessions to stop being verbose.  However, removing the `verbose` in neural/evaluator stops the "test" process to stop being verbose.  Thus, it seemed like conflict handler was the simplest way to handle this.